tensorflow_datasets/image/imagewoof.py

```diff
@@ -55,57 +53,53 @@
-                           'Rhodesian ridgeback', 'Dingo', 'Golden retriever', 'Old English sheepdog']),
+  def _split_generators(self, dl_manager):
+    """Generate Splits"""
+    extracted_path = dl_manager.download_and_extract(_IMAGEWOOF_URL)
+    extracted_path = os.path.join(extracted_path, 'imagewoof2')
+    # Specify the splits
+    return [
+        tfds.core.SplitGenerator(
+            name=tfds.Split.TRAIN,
+            gen_kwargs={
+                "images_dir_path": os.path.join(extracted_path, "train"),
+            }),
+        tfds.core.SplitGenerator(
+            name=tfds.Split.VALIDATION,
+            gen_kwargs={
+                "images_dir_path": os.path.join(extracted_path, "val"),
             }),
-            supervised_keys=("image", "label"),
-            homepage="https://github.com/fastai/imagenette",
-            citation=_CITATION
-        )
-
-    def _split_generators(self, dl_manager):
-        # {TODO}: Downloads the data and defines the splits
-        # dl_manager is a tfds.download.DownloadManager that can be used to
-        # download and extract URLs
-        extracted_path = dl_manager.download_and_extract(_IMAGEWOOF_URL) 
-        extracted_path = os.path.join(extracted_path, 'imagewoof2')
-        # Specify the splits
-        return [
-            tfds.core.SplitGenerator(
-                name=tfds.Split.TRAIN,
-                gen_kwargs={
-                    "images_dir_path": os.path.join(extracted_path, "train"),
-                }),
-            tfds.core.SplitGenerator(
-                name=tfds.Split.VALIDATION,
-                gen_kwargs={
-                    "images_dir_path": os.path.join(extracted_path, "val"),
-                }),
         ]
 
-    def _generate_examples(self, images_dir_path):
-        # {TODO}: Yields (key, example) tuples from the dataset
-        print(images_dir_path)
-        count = 0
-        for image_folder in tf.io.gfile.listdir(images_dir_path):
-            for image_file in tf.io.gfile.listdir(os.path.join(images_dir_path, image_folder)):
-                yield count, {
-                    'image': '{}/{}/{}'.format(images_dir_path, image_folder, image_file),
-                    'label': lbl_dict[image_folder]
-                }
-                count += 1
\ No newline at end of file
+  def _generate_examples(self, images_dir_path):
+    """Generate examples given the image directory path"""
+    count = 0
+    for image_folder in tf.io.gfile.listdir(images_dir_path):
+      for image_file in tf.io.gfile.listdir(os.path.join(images_dir_path,
+                                                         image_folder)):
+        yield count, {
+            'image': '{}/{}/{}'.format(images_dir_path, image_folder,
+                                       image_file),
+            'label': lbl_dict[image_folder]
+        }
+        count += 1
```

Comment:
I don't know why its not showing you error for adding extra one line space after ending of code.